catalogue/webapp/services/prismic/transformers/index.ts

```diff
@@ -1,24 +1,18 @@
-import {
-  PrismicResponse,
-  TransformedResponse,
-  ContentType,
-  Contributor,
-} from '../types';
+import { PrismicResponse, Story, Contributor } from '../types';
 import { articleIdToLabel } from '../fetch';
 import { isNotUndefined } from '@weco/common/utils/array';
 import { transformImage } from '@weco/common/services/prismic/transformers/images';
 import linkResolver from '@weco/common/services/prismic/link-resolver';
 
 export async function transformPrismicResponse(
-  type: ContentType[],
-  edges: PrismicResponse[]
-): Promise<TransformedResponse[]> {
+  edges: PrismicResponse[],
+  type: 'articles' = 'articles'
```

Comment:
Do we still need this `type` argument if it's never getting a non-default value?

Reply:
No, we need to return one, I think (Cause we check if it's 'articles' or 'Error'), but we don't need to pass one. I'll remove.